model/src/pyrenew/process/iidrandomsequence.py

```diff
@@ -0,0 +1,145 @@
+# -*- coding: utf-8 -*-
+# numpydoc ignore=GL08
+
+import numpyro.distributions as dist
+from numpyro.contrib.control_flow import scan
+from pyrenew.metaclass import DistributionalRV, RandomVariable, SampledValue
+
+
+class IIDRamdomSequence(RandomVariable):
```

Comment:
```suggestion
class IIDRandomSequence(RandomVariable):
```
Present throughout, but only commenting here.

Reply:
Oops! Will fix.

Reply:
Fixed in 0e08ee9